Lib/unittest/case.py

```diff
@@ -338,6 +339,19 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
+def _short_description_from_docstring(text):
+    # Return the summary line from a docstring.
```

Comment:
I suggested to replace too verbose docstring in initial commit with shorter comment. But since it was shortened I'm nice with a docstring too.